db/dbConfig.js

```diff
@@ -0,0 +1,4 @@
+const knex = require('knex');
+const knexConfig = require('../knexfile');
+
+module.exports = knex(knexConfig.development);
```

Comment:
Rather than using just development, why not prepare for other environments like testing or production as well? For example:
```javascript
// ...
const env = process.env.DB_ENV || 'development';
module.exports = knex(knexConfig[env]); 
```